package.json

```diff
@@ -129,7 +129,7 @@
     "regenerator-runtime": "^0.13.1",
     "require-reload": "^0.2.2",
     "simple-git": "^1.95.0",
-    "styled-components": "5.1.1",
+    "styled-components": "^5.1.1",
```

Comment:
Thanks @ShimiSun . When I tried this the differences were mostly just classnames, but there were some tests that failed outright when they couldn't find elements they were looking for. Did you run into this or find a solution?

Reply:
Worked like a charm. which version exactly did you try using?
I just pushed a commit that uses 5.2.1. the latest directly to make sure the errors don't appear and don't break anything.

Reply:
Awesome. I ended up with 5.2.0 at the time